app/subscriber/src/features/utils/formatDate.ts

```diff
@@ -9,7 +9,7 @@ export const formatDate = (date: string | Date, includeTime?: boolean): string =
   const result = moment(date);
   if (!result.isValid()) return '';
   if (includeTime) {
-    return result.format('MMMM DD, YYYY hh:mm:ss');
+    return result.format('MMMM DD, YYYY hh:mm:ss a');
```

Comment:
Fixed issue as without the `a` it was impossible to know if it was morning or evening.